dotcom-rendering/src/web/components/TableOfContents.importable.tsx

```diff
@@ -57,7 +85,7 @@ const summaryStyles = css`
 
 const titleStyle = css`
 	${textSans.xsmall({ lineHeight: 'regular' })}
-	color: ${neutral[46]};
+	color: ${neutral[7]};
```

Comment:
same comment as before - can we import neutral7 from palette instead for these 3?